### DISCUSSION TITLE
How do I make regex ignore case or use other regex flags?

### DISCUSSION THREAD

**Discussion by JayPalm** [pydantic/pydantic#3825] [Question]
I want to validate a field with a case insensitive regex pattern. Is there a way to pass `re.I` to the pattern constructor? The constr constructor looks like it only accepts a string, so maybe it could also accept a compiled `typing.Pattern`?

**Comment by ytausch** [comment]
resolved since pydantic 2.8: https://pydantic.dev/articles/pydantic-v2-8-release
